microsoft-edge/devtools-guide-chromium/storage/cookies.md

```diff
@@ -30,16 +30,15 @@ See [Using HTTP cookies](https://developer.mozilla.org/docs/Web/HTTP/Cookies).
 <!-- ====================================================================== -->
 ## Open the Cookies pane
 
-1. [Open DevTools](/microsoft-edge/devtools-guide-chromium/open).
+1. Open DevTools on the webpage you want to edit. To open DevTools, use one of the techniques described in [Open DevTools](/microsoft-edge/devtools-guide-chromium/open).
```

Comment:
```suggestion
1. Open DevTools on the webpage you want to edit.  For example, right-click the page and then select **Inspect**, or press F12.  For additional ways, see [Open DevTools](/microsoft-edge/devtools-guide-chromium/open).
```
Provide at least one way, inline, in order to avoid having to leave the page.

Reply:
Probably delete "For additional ways, see [Open DevTools](/microsoft-edge/devtools-guide-chromium/open)."  Not needed, not the focus.